src/components/figures/chart/chartJs/chartjs.ts

```diff
@@ -1,15 +1,18 @@
 import { Component, onMounted, onWillUnmount, useEffect, useRef } from "@odoo/owl";
-import type { Chart, ChartConfiguration } from "chart.js";
+import { Chart, ChartConfiguration } from "chart.js/auto";
+// import Chart from "chart.js";
```

Comment:
:eyes:

Reply:
Yeah sorry I didn't clean up my mess... Should be better now :sweat: